server/chat-plugins/scavengers.ts

```diff
@@ -1691,6 +1691,7 @@ const ScavengerCommands: ChatCommands = {
 		}
 		if (!target && this.cmd !== 'queuerecycled') {
 			if (this.cmd === 'queue') {
+				if (!this.runBroadcast()) return false;
 				const commandHandler = ScavengerCommands.viewqueue as ChatHandler;
```

Comment:
Is there a reason this doesn't use `this.parse`?
It feels like a better solution would be for it to use that, and to make the regex follow parse calls.
Covers a more general use-case.